sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from "@nestjs/common";
 import { ApplicationService } from "../../services";
 import BaseController from "../BaseController";
-import { GetApplicationBaseDTO } from "./models/application.model";
+import { GetApplicationBaseAPIOutDTO } from "./models/application.dto";
```

Comment:
As a suggestion, we can avoid Verb in DTO names like Get. e.g `ApplicationBaseAPIOutDTO` instead.